docs/dev-guide/web.md

```diff
@@ -7,21 +7,88 @@ sidebar_label: Start
 Welcome to the developers guide for web! This guide will help you setup a development
 environment to start working on the Jitsi Meet codebase.
 
-## Building the sources
+
+## Prerequisites
+
+The setup for the development environment uses `make`, a native command from GNU/Linux. **This guide assumes that you have a Linux system with Ubuntu or Debian, or Windows Subsystem for Linux (WSL)[https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux]**. If you have iOS or other Linux distro, the process should of the installation of packages will be diferent depending on your OS.
```

Comment:
Maybe add a `::: note` for windows users instead? The proposed wording sounds confusing since it doesn't mention macOS for example.